torch/_subclasses/meta_utils.py

```diff
@@ -795,16 +795,30 @@ def empty_create_subclass(
             outer_size = outer_size if outer_size is not None else t.size
             outer_stride = outer_stride if outer_stride is not None else t.stride
 
-            def transform(attr, inner_t):
+            def transform(attr, inner_t, prefix):
+                if is_traceable_wrapper_subclass(inner_t):
+                    return inner_t
+
+                attr_fqn = prefix + "." + attr if prefix != "" else attr
+                attr_list = attr_fqn.split(".")
```

Comment:
If all you're going to do to the attr_fqn is split it, why not just pass around a list